src/tags/object/PagedView.js

```diff
@@ -114,6 +115,11 @@ const HtxPagedView = observer(({ item }) => {
     };
   }, [page]);
 
+  const changePageSize = (pageSize) => {
```

Comment:
Not sure the solution here is to hardcode to 1, can you explain the performance issue with the scrolling to?